test/system/tools/fds/CMakeLists.txt

```diff
@@ -24,21 +24,66 @@ if(PYTHONINTERP_FOUND)
         test_fast_discovery_closure
     )
 
+    # windows auxiliary script to fork test execution
+    set(PWS_LAUNCHER
+        ${CMAKE_CURRENT_SOURCE_DIR}/launcher.ps1
+    )
+
+    # Calculate environment
+    set(TEST_ENVIRONMENT 
+        "PATH=$ENV{PATH};$<TARGET_FILE_DIR:fastcdr>;$<TARGET_FILE_DIR:fastrtps>")
+
+    if(WIN32)
```

Comment:
You could slightly comment why this is necessary.

Reply:
Everybody knows we must populate the environment to help a binary locate its dependencies. That's why we use local_setup.sh or setup.sh in colcon.